### ISSUE TITLE
Plans to merge into bevy?

### ISSUE THREAD

**Issue by jbcurtin** [aevyrie/bevy_mod_raycast#28]
Hi, are there plans to merge this code into bevy?

**Comment by aevyrie** [comment]
Yes, cart is aware of mod_raycast and mod_picking. It simply hasn't been a priority.

I expect this will start to happen once https://github.com/bevyengine/rfcs/pull/12 is merged.
